parl/remote/remote_manager.py

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import queue
+import sys
+if sys.version_info >= (3, 0):
+    import queue as Queue
+else:
+    import Queue
 import threading
```

Comment:
Thanks for your PR：）
I recommend we solve the compatible problem between python2.7 and python3.x in this way:
`from six.moves import queue`
, a more compact solution.
Could you update your modification?